/*
 * utilities.proto - Utilities service protocol definitions.
 * Copyright (C) 2019-2020 University of Texas
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

syntax = "proto3";
option java_multiple_files = false;
option java_package = "org.astria.rpc";
option java_outer_classname = "UtilitiesRequest";

import "messages.proto";

service Utilities
{
    rpc importTDM(ImportTDMInput) returns (Measurement2DArray) {}

    rpc interpolateEphemeris(InterpolateEphemerisInput) returns (MeasurementArray) {}
}
